13_biblioteki/13_8_AlgorithmsBLAS.tex

```diff
@@ -0,0 +1,257 @@
+% !TEX encoding = UTF-8 Unicode
+\section{Algorytmy blokowe i BLAS}
+%%%%%%%%%%%%%%%%
+	\begin{frame}{Faktoryzacja}
+		Level 1, 2, 3 BLAS performance \\
+		Blocked algorithms \\
+		$$
+		\begin{pmatrix}
+		A_{11} & a_j & A_{13} \\
+		. & a_{jj} & a_j^T \\
+		. & . & A_{33}
+		\end{pmatrix}
+		=
+		\begin{pmatrix}
+		U_{11}^T & 0 & 0 \\
+		u_j^T & u_{jj} & 0 \\
+		U_{13}^T & u_j & U_{33}^T
+		\end{pmatrix}
+		*
+		\begin{pmatrix}
+		U_{11} & u_j  & U_{13} \\
+		0 & u_{jj} & u_j^T \\
+		0 & 0 & U_{33}
+		\end{pmatrix}
+		$$
+		Porównując współczynniki w kolumnie j otrzymujemy:
+		$
+		\begin{cases}
+		a_j = U_{jj}^T * u_j \\
+		a_{jj} = u_j^T * u_j + u_{jj}^2
+		\end{cases}
+		$ \\
+		Jeżeli $U_{11}$ został wyznaczony, to $u_j$ oraz $u_{jj}$ można obliczyć z:
+		$
+		\begin{cases}
+		U_{11}^T * u_j = a_j \\
+		u_{jj}^T = a_jj - u_j^T * u_{j}
+		\end{cases}
+		$
+	\end{frame}
+	\begin{frame}[fragile]{Realizacja z użyciem BLAS level 1}
+		\begin{lstlisting}
+			DO 30 J = 1, N
+				INFO = J
+				S = 0.0E0
+				JM1 = J - 1
+				IF(JM1 .LT. 1) GO TO 20
+				DO 10 K = 1, JM1
+					T = A(K,J) - SDOT(K-1, A(1,K), 1,
+					A(1,J), 1)
+					T = T/A(K,K)
+					A(K,J) = T
+					S = S + T*T
+			10 CONTINUE
+			20 CONTINUE
+			S = A(J,J) - S
+			... EXIT
+			IF (S .LE. 0.0E0) GO TO 40 A(J,J) = SQRT(S)
+			30 CONTINUE		
+		\end{lstlisting}
```

Comment:
Ok, then there is the problem. That's how it looks like to me:
![bez nazwy](https://user-images.githubusercontent.com/23387533/29515014-e703735e-866a-11e7-854a-a274a7f61b7d.jpeg)
Then will look for a more reliable way